src/other/WordCloud.js

```diff
@@ -38,7 +38,7 @@
     };
 
     WordCloud.prototype.enter = function (domNode, element) {
-        this.cloud = d3.layout.cloud()
+        this.cloud = new d3Cloud()
```

Comment:
need to add if statement for non-amd version and use d3.layout.cloud otherwise it doesnt render

        if (typeof define === "function" && define.amd) {
            this.cloud = d3Cloud();
        } else {
            this.cloud = d3.layout.cloud();
        }